packages/manager/modules/support/src/index.js

```diff
@@ -1,5 +1,6 @@
 import angular from 'angular';
 
+import ngAtInternet from '@ovh-ux/ng-at-internet';
```

Comment:
Importing peerDependencies in your modules should be done in that way:

```js
import '@ovh-ux/ng-at-internet';
```

It could also be applied for the rest.

Reply:
It's my fault, don't do as I do @frenautvh  😄

Reply:
Hello guys, i just added a commit to remove rollup and makes the main point to src/index.js. It's better that way because the dev building time is reduced a lot.

Now i don't think it's an issue to import peer dep like that since it ony was a problem when the build was targeting a commonJS with rollup. Using webpack it's safe.

What do you think?

Reply:
`rollup.config.js` can also be removed then.

Reply:
If this package turned as a private one it means that you cannot rely on the actual version of the Dedicated Control Panel (ovh-ux/ovh-manager-dedicated I mean).

It clearly depends on the time frame to be able to put the Dedicated Control Panel in production through the monorepo (just to keeping it in mind).

Reply:
> `rollup.config.js` can also be removed then.

done thanks